Use correct package name when looking up version

alcaeus opened the pull request:
Fixes #2510.

Smolevich commented:
@alcaeus master branch has changes with laravel 10, maybe to change destination branch from master to 3.9 ?

Smolevich commented on src/Connection.php:
```diff
@@ -328,7 +328,7 @@ private static function lookupVersion(): string
         if (class_exists(InstalledVersions::class)) {
             try {
-                return self::$version = InstalledVersions::getPrettyVersion('jenssegers/laravel-mongodb');
+                return self::$version = InstalledVersions::getPrettyVersion('jenssegers/mongodb');
```
I suggest don't use assignment in return block, and separate operation assignment and return value

alcaeus replied:
Updated the code accordingly 👍

alcaeus commented:
@Smolevich that's a good point, but there is no 3.9 branch. If you could create one based on the latest 3.9 release and push it I'll rebase and re-target this PR.

Smolevich commented:
> @Smolevich that's a good point, but there is no 3.9 branch. If you could create one based on the latest 3.9 release and push it I'll rebase and re-target this PR.

Let's do it

alcaeus commented:
@Smolevich I can't push the branch myself, but once it's pushed this PR can be changed to target 3.9. I've rebased on top of `v3.9.4` and dropped all commits related to Laravel 10 support that should not be in 3.9.

divine commented:
We need to update the master branch as well.
